index2.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+
+<style>
+
+    .block {
+        display: flex;
+        justify-content: space-around;
+        padding: 100px;
+    }
+    .one {
+        width: 400px;
+        height: 300px;
+        overflow: hidden;
+    }
+    .one-img {
+        width: 400px;
+        height: 300px;
+    }
+    .one-img:hover {
+       transform: scale(0.8);
+       transition: 1s;
+    }
+    .two {
+        width: 400px;
+        height: 300px;
+     
+    }
+    .two-img {
+        width: 400px;
+        height: 300px;
+       
+    }
+    .two-img:hover {
+        zoom: 1.5;
+        transition: 1s;
+    }
+
+</style>
+</head>
+
+<body>
+    <div class="block">
+
+        <div class="one">
+            <img class="one-img" src="pm_01.jpg" alt="">
+        </div>
+    
+        <div class="two">
+        <img class="two-img" src="pm_03.jpg" alt="">
```

Comment:
Чи можна додати alt?
Так само можна додати декларативно розміру тут.
І було б добре змінити ім'я класа на те, яке відповідає семнтиці елемента.

```suggestion
        <img class="hove-effect-demo" width="300" height="200" src="pm_03.jpg" alt="Сілует дитини, яка пригає з дюни, за нею сходить сонце, наповнюючи повітря сріблясто-фіолетовим сяйвом">
```